packages/admin/admin/src/form/file/FileSelect.tsx

```diff
@@ -45,8 +45,8 @@ type OwnerState = {
 export type FileSelectProps<AdditionalValidFileValues = Record<string, unknown>> = {
     files: FileSelectItem<AdditionalValidFileValues>[];
     onDrop?: DropzoneOptions["onDrop"];
-    onRemove?: (file: FileSelectItem<AdditionalValidFileValues>) => void;
-    onDownload?: (file: FileSelectItem<AdditionalValidFileValues>) => void;
+    onRemove?: (file: FileSelectItem<AdditionalValidFileValues> | ErrorFileSelectItem) => void;
```

Comment:
Isn't this already included in FileSelectItem?

Reply:
Sorry, this was supposed to be `ValidFileSelectItem` | `ErrorFileSelectItem`. 
So anything, except loading items can be removed.